pcsx2/DEV9/Win32/resource.h

```diff
@@ -1,27 +1,25 @@
 //{{NO_DEPENDENCIES}}
-// Microsoft Developer Studio generated include file.
-// Used by DEV9linuz.rc
+// fichier Include Microsoft Visual C++.
+// Utilis� par DEV9ghzdrk.rc
 //
-
-#ifndef IDC_STATIC
-#define IDC_STATIC -1
-#endif
-
-#define IDD_CONFIG 101
-#define IDC_NAME 1000
-#define IDC_ETHDEV 1007
-#define IDC_BAYTYPE 1008
-#define IDC_ETHENABLED 1009
-#define IDC_HDDFILE 1010
-#define IDC_HDDENABLED 1011
+#define IDD_CONFDLG                     801
```

Comment:
resource.h has 101 and this used to have 101. Doesn't that conflict?
![image](https://user-images.githubusercontent.com/24227051/98584097-b5448380-22c5-11eb-80fb-4c85c03fa0b5.png)

Reply:
no it doesn't, that's the whole point, keeping it to 101 does conflict with usb, also auto generated